### PR TITLE
[BIM-2430] Added `windowOpen` and `windowClose` events to documentation

### PR DESCRIPTION
**Pull request by sbieserprocore**
# DO NOT MERGE UNTIL 3/31

Added new events `windowOpen` and `windowClose` to our `events` namespace.
image

### CODE REVIEW

File: bim_webviewer/bim_web_viewer.md [hunk 1]
```diff
@@ -1369,6 +1369,37 @@ Fires when a window based tool has changed its position or size.
...
+
+| Field Name | Type | Description |
+| - | - | - |
+| elementId | string | The value of the id attribute on the root DOM element of the window. |
+| toolId | number | The id of the tool the window belongs to. See [Tools](#tools) for further information. |
+
+
+---
+
+### onWindowHide
```
Comment (sbieserprocore) [inline bim_webviewer/bim_web_viewer.md:1390]: Recommend `windowClose`

File: bim_webviewer/bim_web_viewer.md [hunk 2]
```diff
@@ -1369,6 +1369,37 @@ Fires when a window based tool has changed its position or size.
 | modalSize  | object | An object with the keys, `height`, `left`, `top`, `width` to indicate the current position and size of the window. |
 | name       | string | Human readable name of the window based tool that is changing.                                                     |
 
+---
+
+### onWindowOpen
```
Comment (sbieserprocore) [inline bim_webviewer/bim_web_viewer.md:1374]: `windowOpen`

**Comment by sbieserprocore** [comment]
Web Viewer SDK event renaming PR: https://github.com/procore/bim-webviewer/pull/813

File: bim_webviewer/bim_web_viewer.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sbieserprocore) [reply bim_webviewer/bim_web_viewer.md:1374]: PR [HERE](https://github.com/procore/bim-webviewer/pull/813)

**Review by plardin — APPROVED** [review]

**Review by Bates550 — APPROVED** [review]

**Pull request merged**